bundles/org.openhab.ui/web/src/css/app.styl

```diff
@@ -285,3 +285,10 @@ html
 // Disable user dragging of background images in canvas layouts
 .disable-user-drag
   user-drag none
+
+.failure-toast
+  background-color #e64a19 !important
+  font-size 22px !important
```

Comment:
I would really prefer finding a Framework7 CSS variable (https://v5.framework7.io/docs/css-variables) instead of hardcoding a color, also I'm not sure about this 22px font size :sweat_smile: 

There are other options for example a centered toast if the app is effectively unusable when the connection is broken (from https://v5.framework7.io/docs/toast#examples):

![Screenshot from 2023-04-05 21-21-06](https://user-images.githubusercontent.com/2004147/230185030-d4e4e343-b414-4a47-a58b-f3b62ed14b97.png)

Or maybe a Notification (https://v5.framework7.io/docs/notification#examples)?

![Screenshot from 2023-04-05 21-33-10](https://user-images.githubusercontent.com/2004147/230185901-0e644a30-eb4e-4015-98ca-7b94bee85a66.png)

Reply:
This color is the openHAB orange, I have switched to using the `--f7-theme-color` CSS variable instead of hard-coding it.
I have chosen the 22px font size to make the warning larger.

Regarding the centered toast: I really don‘t like its look without having an icon, and it seems that we can‘t have both icon and reload button. A toast at the bottom seemed the visually best solution to me.

I haven‘t really looked into the notifications, but IMO we should not use them and instead „reserve“ them for real notifications when MainUI supports Web Push one day.

Reply:
Thanks for the answer, I don't mind your choices but had to express mine ;)